internal/cmd/keys_test.go

```diff
@@ -88,3 +93,102 @@ func TestKeysAddCmd(t *testing.T) {
 func requestMatches(req *http.Request, method string, path string) bool {
 	return req.Method == method && req.URL.Path == path
 }
+
+func TestKeysListCmd(t *testing.T) {
+	home := t.TempDir()
+	t.Setenv("HOME", home)
+	t.Setenv("USERPROFILE", home) // for windows
+
+	base := time.Date(2022, 1, 2, 3, 4, 5, 6, time.UTC)
```

Comment:
This isn't going to work with our relative date output. I'll change this to not be a fixed date.